### PR TITLE
TSAL information and circuit image

### PR DESCRIPTION
**Pull request by ljordan51**

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
...
+	    Size (length x height x width): & 41x41x32 mm \\ \hline
 	    \end{tabular}
 	    \caption{Parameters of the TSAL}
 	    \label{TSALparameters}
 	\end{table}
 
 \subsubsection{Wiring/cables/connectors}
 %Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.  Include gauge, voltage and temperature rating of wiring used and any fuses or other overcurrent protection used.
-TODO
+The circuitry designed has a TS-controlled optocoupler. When the TS is over 60V, zener diodes with a breakdown voltage of 58V and 3.6V respectably power the photodiode side which switches the mosfet side allowing current to flow from the GLV to power the TSAL. The frequency of the TSAL will be controlled by built in settings to flash at 2Hz.
```
Comment (aehoppe) [inline esf_main.tex]: *respectively

Comment (aehoppe) [reply esf_main.tex]: also MOSFET should probably be capitalized, along with Zener.

Comment (aehoppe) [reply esf_main.tex]: Also reference the figure using \Fref{}

File: esf_main.tex [hunk 2]
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
...
+
+\begin{figure}[h]
+\centering
+\includegraphics[scale=1]{TSAL}
+\caption{Schematic for the TSAL}
+\end{figure}
+
+All connections made by wires will be 20 AWG rated for 600V, 125°C and 7A, while all PCB traces will be a minimum of 12 mil, but nominally 20 mil in width. The TS voltage will be fused to 1A on the power line.
```
Comment (aehoppe) [inline esf_main.tex]: Good!

File: esf_main.tex [hunk 3]
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
...
 
 \subsubsection{Position in car}
 %Provide CAD-renderings showing the relevant parts. Mark the parts in the rendering, if necessary.
-TODO
+The TSAL will be mounted to the underside of the highest point of the main roll hoop, per EV 4.12.3 & 4 using a robust 3D printed (Kevlar reinforced Nylon) bracket integrating the light and necessary wiring. This enclosure has not been designed yet. The PCB will be located in the master switch panel enclosure.
```
Comment (aehoppe) [inline esf_main.tex]: We'll want a CAD rendering for this probably.

**Review by aehoppe — CHANGES_REQUESTED** [review]
Looks good, needs a few changes though. See comments. Also, thanks for picking this up on short notice!

File: esf_main.tex (hunk 1 above)
Comment (ljordan51) [reply esf_main.tex]: do i need to add a label to the figure and/or put anything in the curly braces? or just type \Fref{} wherever i want to reference the figure? and what does Fref actually insert? Text with the the figure number?

File: esf_main.tex (hunk 3 above)
Comment (ljordan51) [reply esf_main.tex]: do i need to request one from someone in mechanical? if so, who? if not, where can i find the rendering?

**Pull request merged**